handler.go

```diff
@@ -33,3 +33,8 @@ type TokenEndpointHandler interface {
 	// the request, this method should return ErrUnknownRequest and otherwise handle the request.
 	HandleTokenEndpointRequest(ctx context.Context, req *http.Request, requester AccessRequester) error
 }
+
+type RevocationHandler interface {
```

Comment:
It would be awesome if you could give a short description what this interface does